src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,81 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int numberOfColumns = 4;
```

Comment:
```suggestion
    private static final int COLUMNS_NUMBER = 4;
```
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names